### ISSUE TITLE
Podcast RSS feed

### ISSUE THREAD

**Issue by strickvl** [abrudz/abrudz.github.io#2]
First off, thank you for making the effort to get the RSS feed in for the podcast! I tried adding it to my podcast player of choice (Overcast) and gave it the feed URL, but it says that there were no audio-only entries in the feed. I'm not sure how the feed is being generated -- it's not in this repo, it seems -- but in any case I'm wondering if it might be possible to address / fix this? At the moment I have no way of subscribing through my podcast app in the usual way.

**Comment by abrudz** [comment]
Yes, I am aware of this situation. I have [some](https://www.reddit.com/r/apljk/comments/yhiwkr/new_podcast_apl_notation_as_a_tool_of_thought_1st/) [browser](https://github.com/furius95/jekyll-podcast/blob/main/README.md) [tabs](https://help.apple.com/itc/podcasts_connect/#/itcb54353390) [open](https://github.com/martinong/podcast-pages) that I hope can help me resolve the issues, but I simply don't have time to look into it at the moment. The RSS feed is actually generated from this repo, by [/_config.yml](https://github.com/abrudz/abrudz.github.io/blob/master/_config.yml) and [/_posts/](https://github.com/abrudz/abrudz.github.io/tree/master/_posts) but the current setup seems to generate a feed that is suitable for e.g. a blog, rather than for podcasts.

It would be wonderful if someone with the know-how could assist me…
